### PR TITLE
feat: Handle invite errors

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
## Description

Returns human readable message to users when we catch an error as they try to:

- Invite collaborators
- invite members
- add trusted service

closes #435
relates: gliff-ai/dominate#538
I created the issue for this in DOMINATE but it turns out that the fix was to be in MANAGE.. You can see the issue [here](https://github.com/gliff-ai/dominate/issues/435
). 
I think because its from a different repo it's not allowing the issue to be linked to the PR. 

## Dependency changes

N/A

## Testing

N/A

## Documentation

N/A

## Migrations (if applicable)

Have any database migrations been committed as part of this pull request?

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New migrations have been committed
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If appropriate, I have bumped any version numbers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +60,7 @@
     "react-router-dom": "==6.0.0-beta.4"
   },
   "dependencies": {
+    "@gliff-ai/style": "^3.0.0",
```
Comment (ChasNelson1990) [inline package.json:63]: I thought we kept this as a peer+dev dependency, so we don't end up bringing multiple versions into DOMINATE?

**Review by ChasNelson1990 — COMMENTED** [review]
I realise you're off now but I'm not 100% certain about the logic going on here... can we chat about it?

File: package.json (hunk 1 above)
Comment (cooper667) [reply package.json:63]: Yea, it definitely should be

File: package.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
     "@babel/preset-env": "^7.16.4",
     "@gliff-ai/eslint-config": "^0.2.0",
     "@gliff-ai/jest-browserstack-automate": "^5.0.0",
-    "@gliff-ai/style": "^2.8.0",
```
Comment (ChasNelson1990) [inline package.json:29]: I think you still want the dev dependency for local development, it just needs to match the peer dependency? Unless @cooper667 knows of a nice way where we define the version once (as peer) but can install peer deps under dev?

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (cooper667) [reply package.json:29]: Yes to having it in dev AND peer. There's no nice way to have both really.

**Review by cooper667 — DISMISSED** [review]

File: src/views/TeamView.tsx [hunk 3]
```diff
@@ -93,18 +94,21 @@ export const TeamView = (props: Props): JSX.Element => {
...
-    });
-
-    if (result) {
-      setInviteEmail("");
-      setInviteMessage("Invite was sent");
-    } else {
-      setInviteMessage("An error happened with the invite");
+    try {
+      const result = await props.services.inviteUser({
+        email: inviteEmail,
```
Comment (SilviaZeta) [inline src/views/TeamView.tsx:99]: I think `inviteUser` should also be changed for this to work as it is? The method has its own try-catch..

File: src/views/TeamView.tsx [hunk 4]
```diff
@@ -93,18 +94,21 @@ export const TeamView = (props: Props): JSX.Element => {
...
+      });
+      if (result) {
+        setInviteEmail("");
+        setInviteMessage("Invite was sent");
+      } else {
+        setInviteMessage("An error happened with the invite");
+      }
```
Comment (SilviaZeta) [inline src/views/TeamView.tsx:106]: Is this second error message necessary?

**Review by cooper667 — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**